lib/conditionals.md

```diff
@@ -5,8 +5,15 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+Conditionals basically control how a program is executed. In Ruby, you assign a variable a value, but then what are you going to do with it? That’s where flow control comes in. Flow control includes conditionals like if statements and elsif statements:
+
+“if” a condition is met (through user input) do this…
+“elsif” allows for additional condition(s) to be met (you can add as many of these as you want)
+ “else” do this if condition(s) not met
+
 
```

Comment:
You cited examples of control flow like `if`, `else`, and `elsif`. What is one other example of control flow where a conditional might be used?